NepaliCalendar/NepaliCalendarTests/DateConverterTests.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  DateConverterTests.swift
+//  NepaliCalendarTests
+//
+//  Created by alok subedi on 01/05/2021.
+//
+
+import XCTest
+import NepaliCalendar
+
+class DateConverterTests: XCTestCase {
+    
+    func test_bSToAD_throwsErrorWhenDayOrMonthIsLessThanOne() {
+        let datesLessThanOne = [(0, 0), (-1, -1), (-10, -10), (1, -1), (-1, 2)]
+        
+        try? datesLessThanOne.forEach { (day, month) in
+            XCTAssertThrowsError(try DateConverter.bSToAD(date: NCDate(day: day, month: month, year: 2078)))
+        }
+    }
+    
+    func test_bSToAd_throwsErrorWhenMonthIsGreaterThan12() {
+        let monthsGreaterThan12 = [13, 24, 100]
+        
+        try? monthsGreaterThan12.forEach { XCTAssertThrowsError(try DateConverter.bSToAD(date: NCDate(day: 1, month: $0, year: 0))) }
```

Comment:
Use test specific readable variable instead of sending value directly to day and year. Suggestion: `var validYear = 2078` and `var validDay = 1`